perf/jet_nfailures.jl

```diff
@@ -35,7 +35,7 @@ using Test
     # inference. By increasing this counter, we acknowledge that
     # we have introduced an inference failure. We hope to drive
     # this number down to 0.
-    n_allowed_failures = 30
+    n_allowed_failures = 31
```

Comment:
What’s the inference failure being introduced? Can you try to fix this if it’s not too much trouble?